### PR TITLE
Magic Link Integration

### PR DESCRIPTION
**Pull request by Royal-lobster**
fixes https://github.com/EveripediaNetwork/issues/issues/271

# Changes
1. Added Custom Magic Link provider for wagmi 

# Links
https://monopedia-ui-git-magic-link-prediqt.vercel.app/login

# Screenshots
screely-1651049369339

### CODE REVIEW

**Comment by Royal-lobster** [comment]
> npm ? lets make sure we use yarn to avoid conflicts

Ohhh i forgot to remove that. used npm by mistake 😅

**Comment by Royal-lobster** [comment]
> is that 0 for a particular reason? otherwise i would suggest to go w 1 that its ethereum as default use case

Ohh thanks i was about to ask about it. i will change it to 1

File: package.json [hunk 1]
```diff
@@ -49,7 +51,9 @@
     "framer-motion": "^5.6.0",
     "graphql": "^15.8.0",
     "graphql-request": "^3.7.0",
+    "i": "^0.3.7",
```
Comment (kesar) [inline package.json:54]: whats this i library¿? :_D

Comment (Royal-lobster) [reply package.json:54]: Ohh i think 'i' got copied twice while i copied npm i command from docs instead of yarn. ill remove that

**Review by kesar — APPROVED** [review]

**Pull request merged**